packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -503,6 +499,186 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
+                ipAddresses.forEach(async (ipAddress) => {
+                  const associatedPlanByIpAddress = await ipSpendingPlanRepository.findByAddress(
+                    ipAddress,
+                    requestDetails,
+                  );
+                  expect(associatedPlanByIpAddress.planId).to.eq(expectedNonBasicPlans[key].id);
+                  expect(associatedPlanByIpAddress.ipAddress).to.eq(ipAddress);
+                });
+              });
+            });
+          });
+
+          describe('PRIVILEGED Tier', () => {
+            let priviledgedEvmAccount: AliasAccount;
+            let privilegedHbarSpendingPlan: IDetailedHbarSpendingPlan;
+
+            beforeEach(async () => {
+              const result = await createAliasForNonBasicPlans(SubscriptionTier.PRIVILEGED);
+              priviledgedEvmAccount = result.aliasAccount;
+              privilegedHbarSpendingPlan = result.hbarSpendingPlan;
+            });
+
+            afterEach(async () => {
+              await hbarSpendingPlanRepository.delete(privilegedHbarSpendingPlan.id, requestDetails);
+              await ethAddressSpendingPlanRepository.delete(priviledgedEvmAccount.address, requestDetails);
+            });
+
+            it('Should successfully add spending plans', async () => {
+              const plan = await ethAddressSpendingPlanRepository.findByAddress(
+                priviledgedEvmAccount.address,
+                requestDetails,
+              );
+
+              expect(plan.ethAddress).to.eq(priviledgedEvmAccount.address);
+              expect(plan.planId).to.eq(privilegedHbarSpendingPlan.id);
+
+              const spendingPlan = await hbarSpendingPlanRepository.findByIdWithDetails(plan.planId, requestDetails);
+              expect(spendingPlan.active).to.be.true;
+              expect(spendingPlan.amountSpent).to.eq(0);
+              expect(spendingPlan.subscriptionTier).to.eq(SubscriptionTier.PRIVILEGED);
+            });
+
+            it('Should increase the amount spent by the privileged spending plan', async () => {
+              const contract = await deployContract(largeContractJson, priviledgedEvmAccount.wallet);
+
+              // awaiting for HBAR limiter to finish updating expenses in the background
+              await Utils.wait(6000);
+
+              const spendingPlan = await hbarSpendingPlanRepository.findByIdWithDetails(
+                privilegedHbarSpendingPlan.id,
+                requestDetails,
+              );
+
+              const expectedCost = await getExpectedCostOfLastLargeTx(contract.deploymentTransaction()!.data);
+              const amountSpent = spendingPlan.amountSpent;
+
+              expect(amountSpent).to.be.approximately(expectedCost, 0.009 * expectedCost);
+            });
+
+            it('Should eventually exhaust the hbar limit for PRIVILEDGED user', async () => {
+              const fileChunkSize = Number(ConfigService.get('FILE_APPEND_CHUNK_SIZE')) || 5120;
```

Comment:
The default value (`5120`) should be specified inside `globalConfig.ts` so that it's automatically picked up by the `ConfigService`, we should not specify defaults here or anywhere else.

Reply:
The default value of this var in globalConfig.ts was set to `null`, actually. 
```typescript
FILE_APPEND_CHUNK_SIZE: {
      envName: 'FILE_APPEND_CHUNK_SIZE',
      type: 'number',
      required: false,
      defaultValue: null,
    },
```

> we should not specify defaults here or anywhere else
I agree but gotta follow what it was implemented for now.

Reply:
Most likely this was missed in the env refactor PR which adds the ConfigService, 

Let's open a new issue to fix this, cause it was most likely missed not only for this one but for some other variables as well, and `globalConfig.ts` should be the single source of truth for default env values.